### PR TITLE
LF Spec: fix ordering of contract IDs.

### PR DESCRIPTION
**Pull request by remyhaemmerle-da**
CHANGELOG_BEGIN
CHANGELOG_END

### Pull Request Checklist

- [ ] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/main/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [ ] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: daml-lf/spec/daml-lf-1.rst [hunk 1]
```diff
@@ -284,10 +284,8 @@ strings that are part of the syntax with single quotes. We do not
...
 brevity and readability.
 
-
-Literals
-~~~~~~~~
-
+Identifiers
+~~~~~~~~~~~
 In this section, we define a bunch of literals that can be handled by
 DAML-LF programs.
```
Comment (sofiafaro-da) [inline daml-lf/spec/daml-lf-1.rst:290]:
```suggestion
In this section, we define the sorts of strings and identifiers that appear in 
DAML-LF programs.
```

File: daml-lf/spec/daml-lf-1.rst [hunk 2]
```diff
@@ -3233,7 +3232,7 @@ updates.
 
     —————————————————————————————————————————————————————————————————————— EvLessEqTimestamp
       𝕆('LESS_EQ' @σ LitTimestamp₁ LitTimestamp₂) =
-          Ok (LitTimestamp₁ ≤ LitTimestamp₂)
+          Ok (LitTimestamp₁ ≤ₗ≤ LitTimestamp₂)
```
Comment (sofiafaro-da) [inline daml-lf/spec/daml-lf-1.rst:3235]:
```suggestion
          Ok (LitTimestamp₁ ≤ₗ LitTimestamp₂)
```

**Review by sofiafaro-da — APPROVED** [review]
LGTM, thanks!

File: daml-lf/spec/daml-lf-1.rst [hunk 3]
```diff
@@ -3243,7 +3242,10 @@ updates.
 
     —————————————————————————————————————————————————————————————————————— EvLessEqNumeric
       𝕆('LESS_EQ' @σ LitNumeric₁ LitNumeric₂) =
-          Ok (LitNumeric₁ ≤ LitNumeric₂)
+          Ok (LitNumeric₁ ≤ₗ LitNumeric₂)
+
```
Comment (remyhaemmerle-da) [inline daml-lf/spec/daml-lf-1.rst:3246]:
```suggestion
```

**Pull request merged**